build.psm1

```diff
@@ -489,9 +489,15 @@ Fix steps:
     }
 
     # handle TypeGen
-    if ($TypeGen -or -not (Test-Path "$PSScriptRoot/src/System.Management.Automation/CoreCLR/CorePsTypeCatalog.cs")) {
+    $incFileName = 'powershell.inc'
+    if (-not [string]::IsNullOrEmpty($Runtime)) {
+        # File name must be different for Windows and Linux to allow build on Windows and WSL.
+        # Append the first 3 characters fo the Runtime only because otherwise dotnet will crash due to the separation characters. The underscore is only for better readability.
+        $incFileName = "$($incFileName)_$($Runtime.Substring(0,3)).inc"
```

Comment:
Maybe just use `powershell_$($Runtime.Substring(0,3)).inc`? Otherwise it would beome `powershell.inc_xxx.inc`

Reply:
This change has now been undone since we do not need that logic any more due to the usage of `$Options.Runtime`